front/src/modules/timesStops/hooks/useTimesStopsColumns.tsx

```diff
@@ -31,6 +32,22 @@ const timeColumn = (isOutputTable: boolean) =>
     isCellEmpty: ({ rowData }) => !rowData,
   }) as Partial<Column<TimeExtraDays | undefined, string, string>>;
 
+function readOnlyTimeColumn(key: string, dateTimeLocale: Intl.Locale) {
```

Comment:
keep it in the same place as disabledTextColumn, perhaps the 2 column helpers in a new file ?